utils/api/postcodeAPI.spec.ts

```diff
@@ -1,10 +1,15 @@
-import { normalizeAddress } from './postcodeAPI';
+import { formatAddress, lookupPostcode } from 'utils/api/postcodeAPI';
+import axios from 'axios';
+import { addressAPIWrapperFactory } from 'factories/postcode';
+
+jest.mock('axios');
+const mockedAxios = axios as jest.Mocked<typeof axios>;
 
 describe('postcodeAPI', () => {
   describe('normalizeAddresses', () => {
-    it('should work properly', () => {
+    it('should format address data', () => {
```

Comment:
```suggestion
    it('combines the address lines into single line', () => {
```

Definitely an improvement on "should work properly" but maybe this suggestion or something similar is more explicit to what we mean by formatting?